action.yml

```
@@ -1,33 +1,33 @@
-name: 'SVGO action'
-description: 'Automatically run SVGO with GitHub Actions'
-author: 'Eric Cornelissen'
+name: SVGO action
+description: Automatically run SVGO with GitHub Actions
+author: Eric Cornelissen
 
 inputs:
   repo-token:
-    description: 'The GITHUB_TOKEN secret'
+    description: The GITHUB_TOKEN secret
   dry-run:
-    description: 'Run the action in dry mode (i.e. do not write optimized SVGs)'
+    description: Run the action in dry mode (i.e. do not write optimized SVGs)
   ignore:
-    description: 'A glob of SVGs that should not be optimized'
+    description: A glob of SVGs that should not be optimized
   strict:
-    description: 'Fail the Action run even in the event of a recoverable error'
+    description: Fail the Action run even in the event of a recoverable error
   svgo-config:
-    description: 'The path of the configuration file for SVGO'
+    description: The path of the configuration file for SVGO
   svgo-version:
-    description: 'The (major) SVGO version to be used (`1` or `2`)'
+    description: The (major) SVGO version to be used (`1` or `2`)
 
 outputs:
   DID_OPTIMIZE:
-    description: 'Boolean indicating if any SVGs were optimized'
+    description: Boolean indicating if any SVGs were optimized
   OPTIMIZED_COUNT:
-    description: 'The number of SVGs that were optimized'
+    description: The number of SVGs that were optimized
   SVG_COUNT:
-    description: 'The number of SVGs that were detected '
+    description: "The number of SVGs that were detected "
```

The trailing space isn't necessary:

```suggestion
    description: The number of SVGs that were detected
```